src/python_be.h

```diff
@@ -261,6 +261,8 @@ class ModelInstanceState : public BackendModelInstance {
   std::vector<intptr_t> closed_requests_;
   std::mutex closed_requests_mutex_;
 
+  std::thread log_monitor_;
+  bool log_thread_;
```

Comment:
@Tabrizian can this handling be moved to backend level? All Python stubs will send logs to the same queue and avoid spanning additional thread per instance.